app/client.py

```diff
@@ -0,0 +1,56 @@
+# client.py
+
+from app.bucketlist import bluBucketList
+
+class bluClient(object):
+    def __init__(self, userName, userEmail, userPassword):
+        self.userName = userName
+        self.userEmail = userEmail
+        self.userPassword = userPassword
+        self.bL_list = {}
+        self.followedLists = []
+
+
+    def createBList(self, list_name, list_year, list_month, list_quote, set_active):
+        for key,obj in self.bL_list.items():
+            if list_name == key:
+                return False
+        
+        #verifying that the month is not a number:
+        if not list_month.isalpha():
+            return False
+
+        new_bList = bluBucketList(list_name, list_year, list_month, list_quote, set_active)
+        self.bL_list.update({list_name:new_bList})
+        return new_bList
+    
+    def viewList(self, listname):
+        for key,obj in self.bL_list.items():
```

Comment:
Renamed all methods to be descriptive of their purpose with an underscore separating words.
Added docstrings describing every function and purpose of return values where needed.